### DISCUSSION TITLE
How configure Starboard Operator scanner to pull image from Azure (ACR) private registry

### DISCUSSION THREAD

**Discussion by innops** [aquasecurity/starboard#401] [Questions & Help] [answered]
Hi, 
I was wondering how would be possible to configure the Starboard Operator scanner on Helm chart to pull the image from Azure Private Registery? 

It works perfectly when using something like Nginx from public registry, but I'm getting this error when it supposed to pull the image from myregistry.azurecr.io
`unable to initialize a scanner: unable to initialize a docker scanner: 2 errors occurred:\n\t* unable to inspect the image (myregistry.azurecr.io/my-app:v1): Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?\n\t* GET https://myregistry.azurecr.io/oauth2/token?scope=repository%3Amy-app%3Apull&service= myregistry.azurecr.io: UNAUTHORIZED: authentication required`

**Comment by danielpacak** [comment, marked as answer]
@maati55 We do not support Azure (ACR) registry nor we document how to use the Helm chart to enable such integration. We have an open issue to follow up on Azure (ACR) #278 though. As a hint to get started I'd mention that the operator and all scan jobs are run with the service account named by default `starboard-operator`. I don't know if there's a way to authenticate a particular service account or created an image pull secret so this is something to explore in scope of #278.
